indexer/cli/cli.go

```diff
@@ -20,16 +25,29 @@ type Cli struct {
 
 func runIndexer(ctx *cli.Context) error {
 	configPath := ctx.String(ConfigFlag.Name)
-	conf, err := config.LoadConfig(configPath)
+	cfg, err := config.LoadConfig(configPath)
+	if err != nil {
+		return err
+	}
 
-	fmt.Println(conf)
+	// setup logger
+	cfg.Logger = log.New()
+	cfg.Logger.SetHandler(log.LvlFilterHandler(log.LvlInfo, log.StreamHandler(os.Stdout, log.TerminalFormat(true))))
 
+	indexer, err := indexer.NewIndexer(cfg)
 	if err != nil {
-		log.Crit("Failed to load config", "message", err)
+		return err
 	}
 
-	// finish me
-	return nil
+	signalChannel := make(chan os.Signal, 1)
```

Comment:
I believe there is a helper inside of `op-service` for this functionality

Reply:
ah i see it now in opio

Reply:
before fixing this i'm going to put up a quick pr so that the indexer doesn't consume the monorepo as a dependency. Currently not recognizing the op-service dir.

Also explains why the bindings update for CrossDomainMessenger & StandardBridge weren't showing

Reply:
updated as well as utilize op-service/log for cli flags